ring.go

```diff
@@ -132,6 +139,13 @@ func (opt *RingOptions) getPassword(shard string) string {
 	return opt.Password
 }
 
+func (opt *RingOptions) getTLSConfig(shard string) *tls.Config {
+	if opt.TLSConfig != nil {
```

Comment:
TLSConfigs?

Reply:
@vmihailenco This is following the same kinda setup as the password one. If there's a single config at the toplevel, use that, otherwise check the shards config. There it's also called `opt.Password` vs. `opt.Passwords` for the shard list.

Reply:
Then it should be `opt.TLSConfig == nil`

Reply:
@vmihailenco Of course, fixing that.